packages/client/lib/rpc/modules/engine.ts

```diff
@@ -689,6 +747,13 @@ export class Engine {
     }
   }
 
+  async getPayloadV1(params: [string]) {
+    return this.getPayload(params)
+  }
+
+  async getPayloadV2(params: [string]) {
+    return this.getPayload(params)
+  }
   /**
    * Compare transition configuration parameters.
    *
```

Comment:
Comment for the whole file:

can't completely oversee for now so just to make sure: are these error cases for `newPayloadV2` and `fcuV2` from here https://github.com/ethereum/execution-apis/pull/195/files#diff-571851156237b81c20006ce9d7a70b3421f19540cead91f45955316c52fe94c0R367 (e.g.) integrated? 

So the checks on the withdrawal functionality being activated or not together with the `INVALID` responses?

Reply:
would like to do a followup PR on the same (and merge this first) if thats ok? :slightly_smiling_face: